src/components/AppController/AppController.js

```diff
@@ -23,13 +23,13 @@ var App = React.createClass({
       <Grid fluid>
         <Row>
           <Col lg={4} md={4}>
-            <Selector label={"Model Selection"} onChange={this.updateSelection.bind(this, 'model_id')} items={this.getMetadataItems('model_id')}/>
+            <Selector label={"Model Selection"} onChange={this.updateSelection.bind(this, 'model_id')} items={this.getMetadataItems('model_id')} value={this.state.model_id ? this.state.model_id : null}/>
```

Comment:
Hm, looks like I don't. `Selector` itself converts falsy `props.value` inputs to `undefined`, no need to repeat that.

Which does means we can't display a `model_id` of 0, but `model_id`s are strings, so something has already gone pretty badly if it is being attempted.